### DISCUSSION TITLE
SUEWS advanced

### DISCUSSION THREAD

**Discussion by PasqBr** [UMEP-dev/UMEP#261] [Q&A] [answered]
Hello,

i nearly finished the SUEWS tutorial of New York - everything worked perfectly until running SUEWS Advanced. The following error occured:

[Errno 2] File C:\Users\...\Documents\Data\SUEWS NY\SUEWSSpatial_Tutorialdata\suews_prepare\SUEWS_AnthropogenicEmission.txt does not exist: 'C:\\Users\\...\\Documents\\Data\\SUEWS NY\\SUEWSSpatial_Tutorialdata\\suews_prepare\\SUEWS_AnthropogenicEmission.txt'

Check: C:\Users\...\Documents\supy.log

I checked supy.log file and the following text was written:

SuPy - INFO - All cache cleared. 
Actually I did not clear anything...

Anyone knows how to solve this problem?

Cheers

**Comment by biglimp** [comment, marked as answer]
Hm, this might be a bug you have found. 
What version of UMEP and SuPy are you using. Are tou using Windows? The SUPY version should be displayed in the General-tabin the log-section (speech bubble lower right) in QGIS.

**Reply by PasqBr** [reply]
UMEP is the newest version 3.17 and SuPy I update yesterday as well to the current version (I don't remember the number of the version and today it's not possible to check it - is it urgently necessary to solve the problem? then I will check tomorrow.)

**Reply by biglimp** [reply]
Ok no hurry. Check tomorrow and report

**Reply by PasqBr** [reply]
SuPy Version installed: 2021.2.26

**Reply by sunt05** [reply]
It seems that input table is missing after your SUEWS prepare step. Can you please double check if that file exists? And what files are actually there? i.e. under `C:\Users...\Documents\Data\SUEWS NY\SUEWSSpatial_Tutorialdata\suews_prepare\`

If nothing there, we may need to look into the SUEWS prepare plug-in for the cause.

**Reply by PasqBr** [reply]
i have 3 files:
- initial conditionsnyc_2010.nml 
- nyc_2010_data_60 (txt)
- SUEWS_Site_Select (txt)
if i want to select the input folder in suews advanced (Suews prepare) where i saved those three files (output of suews prepare) no files are listed in that folder although they exist...

**Reply by biglimp** [reply]
This file is retrieved from other places and should be located there. Let me check somethings and come back to you later

**Reply by biglimp** [reply]
I cannot replicate this error. The file **SUEWS_AnthropogenicEmission.txt** is not missing after running SUEWSPRepare. Try to update to latest UMEP and please try again. 

![image](https://user-images.githubusercontent.com/11908356/112852117-e8888300-90ab-11eb-830f-44c1d3911f0c.png)

**Reply by PasqBr** [reply]
thanks for checking the error. that's strange - i will update and try again as soon as i am back to working place in a week and let you know (only if i still have issues).
